pkg/redist/System.Collections.Concurrent/lib/System.Collections.Concurrent.builds

```diff
@@ -0,0 +1,9 @@
+<?xml version="1.0" encoding="utf-8"?>
```

Comment:
All of these are point in time things that we ship once.  I don't want to litter src with them.